Dockerfile.special-resource-operator-build

```diff
@@ -0,0 +1,17 @@
+FROM registry.ci.openshift.org/openshift/release:golang-1.16
+
+ENV GO111MODULE=on
+ENV GOFLAGS=""
+
+RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
+RUN chmod 700 get_helm.sh
+RUN ./get_helm.sh
+RUN yum install -y which
+RUN yum install -y podman docker
```

Comment:
Is it working correctly? I know that in RHEL those 2 aren't working well together :) 
Why not `podman` and `podan-docker` instead?

Reply:
i don't see any difference in those.This is the same setup we use in AI, and it works there